RangeBarSample/src/main/java/com/appyvet/rangebarsample/RecyclerViewActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.appyvet.rangebarsample;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+
+public class RecyclerViewActivity extends Activity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        // Removes title bar and sets content view
+        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
+        setContentView(R.layout.activity_recyclerview);
+        RecyclerView recyclerView = findViewById(R.id.recyclerView);
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
+        recyclerView.setAdapter(new Adapter());
```

Comment:
Please add the following lines so that state/progress of rangebar is kept while scrolling the recyclerview 
```
Adapter adapter=new Adapter();
        adapter.setHasStableIds(true);
        recyclerView.setAdapter(adapter);
```

Reply:
Done.